packages/react-components/react-toolbar/etc/react-toolbar.api.md

```diff
@@ -34,7 +34,7 @@ export const ToolbarButton: ForwardRefComponent<ToolbarButtonProps>;
 
 // @public
 export type ToolbarButtonProps = ComponentProps<ButtonSlots> & Partial<Pick<ButtonProps, 'disabled' | 'disabledFocusable'>> & {
-    appearance?: 'primary' | 'subtle';
+    appearance?: 'primary' | 'secondary' | 'subtle';
```

Comment:
The `secondary` appearance works by accident as the `ToolbarButton` renders `Button` under the hood.
I guess the same way `outline` or `transparent` would work.
I do not see any mention of the `secondary` appearance in a design spec. If we need to support it, the design spec should be updated as well.

Reply:
I see, shall I just close this pr then?